src/app/pages/HomePage/HomePage.tsx

```diff
@@ -11,24 +11,34 @@ import {
 import Curation from '../../components/Curation';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import styles from './index.styles';
+import MetadataContainer from '../../components/Metadata';
 
 interface HomePageProps {
   pageData: {
     pageType: string;
     id?: string;
     title: string;
     curations: CurationData[];
+    description?: string;
   };
 }
 
 const HomePage = ({ pageData }: HomePageProps) => {
-  const { curations } = pageData;
-  const { translations, product, serviceLocalizedName } =
+  const { curations, description } = pageData;
```

Comment:
Let's do this, so we can remove the `description: string | undefined;` in src/app/components/Metadata/types.ts

```suggestion
  const { curations, description = '' } = pageData;
```